tests/StackExchange.Redis.Tests/PubSub.cs

```diff
@@ -535,7 +535,8 @@ public async Task PubSubGetAllCorrectOrder_OnMessage_Async()
                             Monitor.PulseAll(syncLock);
                         }
                     }
-                    return i % 2 == 0 ? null : Task.CompletedTask;
+                    // TODO: What the freaking hell was this ever trying to do?
+                    return i % 2 == 0 ? null! : Task.CompletedTask;
```

Comment:
I...wut?

Reply:
cc @mgravell ^

Reply:
It's a test so... two scenarios for the price of one??